main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -60,6 +60,8 @@ object DocAst {
 
     case class Tag(sym: Symbol.CaseSym, args: List[Expr]) extends Atom
 
+    case class ExtTag(label: Name.Label, args: List[Expr]) extends Atom
```

Comment:
hmm you could just use a Keyword combined with a Tag but then the tag has to be refactored not to use casesym directly

Reply:
But if ExtTag is added, it should be composite